pager.c

```diff
@@ -203,6 +203,10 @@ struct PagerRedrawData
 
 /* hack to return to position when returning from index to same message */
 static int TopLine = 0;
+
+
+/* The amount of lines a signature takes */
+static const unsigned int NUM_SIG_LINES = 4;
```

Comment:
Can we make it local to the function that uses it?

Reply:
done